asr-improve-recognition-for-specific-words.md

```diff
@@ -22,11 +22,11 @@ To improve the recognition of specific words, use the following customizations.
-| 5. Fine tune an existing acoustic model | Moderately hard | Fine tune an existing acoustic model using a small amount of domain data to better suit the domain. | When transcribed domain audio data is available (10h-100h), and other easier approaches fall short.   | [Citrinet - Training Tutorial](asr-python-advanced-finetune-am-citrinet-tao-finetuning.ipynb), [Citrinet - Deployment Tutorial](asr-python-advanced-finetune-am-citrinet-tao-deployment.ipynb),<br> [Jasper and Quartznet](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/tao/resources/speechtotext_notebook) |
+| 1. Word boosting                        | Quick and easy  | Temporarily extend the vocabulary while increasing the chance of recognition for a provided list of words.              | When you know that certain words or phrases are important.                                 | [Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-wordboosting.ipynb) |
+| 2. Custom vocabulary                    | Easy            | Permanently extend the default vocabulary to cover novel words of interest.                                             | When the default model vocabulary does not sufficiently cover the domain of interest.      | [Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-customize-vocabulary-and-lexicon.ipynb) |
+| 3. Custom pronunciation (Lexicon mapping)                      | Easy            | Explicitly guide the decoder to map one or more pronunciations (sequences of tokens) to a specific word                                          | When you know a word can have one or several pronunciations.                            | [Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-customize-vocabulary-and-lexicon.ipynb) |
+| 4. Retrain language model               | Moderate        | Train a new language model for the application domain to improve the recognition of domain specific terms.     | When domain text data is available.                        | [Training Tutorial]() |
+| 5. Fine tune an existing acoustic model | Moderately hard | Fine tune an existing acoustic model using a small amount of domain data to better suit the domain. | When transcribed domain audio data is available (10h-100h), and other easier approaches fall short.   | [Citrinet - Training Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-citrinet-nemo.ipynb), [Citrinet - Deployment Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-deployment-citrinet.ipynb),<br> [Conformer-CTC - Training Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-finetuning-conformer-ctc-nemo.ipynb), [Conformer-CTC - Deployment Tutorial](https://github.com/nvidia-riva/tutorials/blob/main/asr-deployment-conformer-ctc.ipynb) |
```

Comment:
The above tutorial links result in a 404 error (except for "Citrinet - Deployment Tutorial" and "Conformer-CTC - Training Tutorial"). I'm assuming once we go live, these tutorials will also.

Reply:
Some of the NeMo-based tutorials still need to be created (I haven't seen PRs for them), but yes